amida-tech/indaba-client
# Bug/INBA-721 File in review stage

#### What does this PR do?
Shows an uploaded file name in non-respondable surveys instead of hiding the inputs entirely

#### Related JIRA tickets:
[INBA-721](https://jira.amida-tech.com/browse/INBA-721)

#### How should this be manually tested?
1. Make a workflow with complete and review stages and a survey with a question that takes file attachments, and assign the tasks
1. Upload a file to the question as the assigned user and complete the task
1. Log in as the user assigned to the review stage and check that the file name is displayed under that question, but cannot be removed
1. Check that viewing the survey as a non-assigned admin also displays the filename
1. Check that viewing the completed task as the user who completed it also shows the filename

#### Background/Context

#### Screenshots (if appropriate):
